### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by FredChauviere**
Minor and major corrections ("International")

### CODE REVIEW

File: locale/fr.json [hunk 1]
```diff
@@ -236,24 +236,24 @@
...
 	"XK": "Kosovo",
 	"YE": "Yemen",
 	"YT": "Mayotte",
 	"ZA": "Afrique du Sud",
 	"ZM": "Zambie",
 	"ZW": "Zimbabwe",
-	"ZZ": "Internacional"
-}
\ No newline at end of file
+	"ZZ": "International"
```
Comment (catamphetamine) [inline locale/fr.json]: Are you sure it's not "Internationale"?

Comment (FredChauviere) [reply locale/fr.json]: Yep. 
I’m a french speaker. Here’s an example in context : https://www.voyagevoyage.ca/conseils/application-appeler-international-pas-cher-1.2487299

But it’s definitely not "Internacionale". All the other corrections come directly from Wikipedia Fr or my own knowlegde (backed by Wikipedia)

Comment (catamphetamine) [reply locale/fr.json]: @FredChauviere Ok, merging.

Comment (FredChauviere) [reply locale/fr.json]: But for the rest, awesome job! Thank you!

**Comment by catamphetamine** [comment]
Released `react-phone-number-input@2.3.22`

**Pull request merged**